en.md

```diff
@@ -0,0 +1,230 @@
+
+The protection of your privacy is extremely important to us. This data protection declaration provides information on how and for what purpose we process personal data and explains the rights to which you are entitled. We are aware of how important personal data is to our users and we uphold all German and European data protection provisions.
+
+## 1. Contact & data protection officer
+In relation to the General Data Protection Regulation (DSGVO), the organisation responsible is: 
+
+Ecosia GmbH  
+Schinkestraße 9  
+12047 Berlin Germany  
+[info@ecosia.org](mailto:info@ecosia.org)
+
+Our Data Protection Officer can be contacted at privacy@ecosia.org or our postal address with the addition of “data protection”.
```

Comment:
In the german version, we used the lower double quote - higher double symbols pair. Is it expected to be different in english?